interview-questions.md

```diff
@@ -7,48 +7,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: It is a snapshot of the object in a class
 
-  Researched answer:
+  Researched answer: The instance variable is an object in a class with a local scope. 
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: I assumed it was a block of code in Ruby. 
```

Comment:
So if we look at these interview questions as literally someone standing in front of you asking "What is a block in Ruby?", I think a better answer here would be "I think it has something to do with a block of code in Ruby, but I'll need to look into it some more to make sure. I'd research in the Ruby documentation to verify my knowledge."

This would show an intrinsic pro-activeness to solution finding, while at the same time being honest about your current knowledge. 

Just a tip! :)